From 0b5c5d2d9a3fec0d6db2e530c2535fa46ae023b2 Mon Sep 17 00:00:00 2001
From: Philipp Kerling <yol@casix.org>
Date: Thu, 16 Apr 2020 20:00:21 +0200
Subject: [PATCH] Remove F-Droid badge from README.md

F-Droid is unfortunately unmaintained and ships a version half a year
old, so it is difficult to feature it prominently on the front page of
the repository.
---
 README.md | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/README.md b/README.md
index c1aeffbf1f1b..434f57fb31df 100644
--- a/README.md
+++ b/README.md
@@ -25,9 +25,6 @@
 <a href="https://play.google.com/store/apps/details?id=org.xbmc.kodi" target="_blank">
   <img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" height="80"/>
 </a>
-<a href="https://f-droid.org/en/packages/org.xbmc.kodi/" target="_blank">
-  <img src="https://f-droid.org/badge/get-it-on.png" height="80"/>
-</a>
 
 <h1 align="center">
   Welcome to Kodi Home Theater Software!
